tests/metrics/network/network-metrics-memory-1g.sh.in

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures Proportional Set Size memory while running an
+#  inter (docker<->docker) 1 Gb tranfer rate using nuttcp.
+#  The selection of 1 Gb as a tranfer rate is because that is 
+#  the maximum that we can be handle in our testing infrastructure.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Currently default nuttcp has a bug
+# see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=745051
+# Image name
+image=gabyct/nuttcp
+
+# We wait for the test system to settle into a steady mode before we
+# measure the PSS. Thus, we have two times - the length of the time the 
+# test runs for, and the time after which we sample the PSS
+
+# Time for the test to run (seconds)
+total_time=6
+# Time in which we sample the PSS (seconds)
+middle_time=3
+
+# Rate limit (speed at which transmitter send data, megabytes)
+# We will measure PSS with a specific transfer rate
+rate_limit=10000
```

Comment:
Hi @GabyCT Thanks for the fixups. I'm still not sure if this 10,000 rate_limit equates to 1Gb(/s) - if it were 1000, then that would make sense (as we are measuring megabytes), or if we were aiming for 1Gbit, and this were 100 Mbytes (taking 1byte on the wire as ~10 bits....), I could understand. So, either I think we may have a typo here (10,000 instead of 1000?), or can we have some more explanation or correction of the text - just so the 1Gb and the 10,000 correlate? :-)  Thanks.